src/modules/sensors/sensor_params.c

```diff
@@ -3009,6 +3009,22 @@ PARAM_DEFINE_INT32(SENS_EN_LL40LS, 0);
 PARAM_DEFINE_INT32(SENS_EN_SF0X, 0);
 
 /**
+ * Lightware SF1xx laser rangefinder
+ *
+ * @reboot_required true
+ * @min 0
+ * @max 4
+ * @group Sensor Enable
+ * @value 0 Disabled
+ * @value 1 SF10/a (25m)
```

Comment:
probably can leave off the (25m) because it seems you are also setting conversion interval